apps/ui/src/main.ts

```diff
@@ -7,6 +7,45 @@ import router from '@/router';
 import '@/helpers/auth';
 import '@/style.scss';
 
+// Intercom
```

Comment:
- This should be in a separated file
- app_id should be a config var 'VITE_INTERCOM_APP_ID"
- If config var is not set we should't load the script
- Messenger should be disabled on small screen

Reply:
Now we are not loading messenger if the screen size is [less than 544](https://github.com/snapshot-labs/sx-monorepo/blob/add-intercom/apps/ui/tailwind.config.js#L55) but this is only on initial load, let me know if we should hide it on resize too (which I think is not needed)